src/consumer/bit-map/bit-map.js

```diff
@@ -184,17 +184,17 @@ export default class BitMap {
     compilerFilesPaths: PathLinux[] = [],
     testerFilesPaths: PathLinux[] = []
   ) {
-    const res = {
+    const ignoreList = {
       files: [],
       dirs: []
     };
-    if (!configDir) return res;
+    if (!configDir) return ignoreList;
```

Comment:
consider using curly brace

Reply:
actually, in this case, a one line is more readable than three lines.
You can't do the mistake of adding another statement to the `if` because of the `return`.

Reply:
Some people read, some people scan code by structure. This will break that practice.